providers/provider_default.go

```diff
@@ -145,3 +147,38 @@ func (p *ProviderData) ValidateSessionState(s *sessions.SessionState) bool {
 func (p *ProviderData) RefreshSessionIfNeeded(s *sessions.SessionState) (bool, error) {
 	return false, nil
 }
+
+func (p *ProviderData) CreateSessionStateFromBearerToken(rawIDToken string, idToken *oidc.IDToken) (*sessions.SessionState, error) {
+	// NOTE: The token is presumably the ID token
```

Comment:
When would it not be the ID token?

Reply:
This was my guess based on my limited knowledge of the code and oauth. Since you have just confirmed that it always is an ID token, I will modify the comment accordingly. Or actually delete it, since the name of the token already says that.